### PR TITLE
Feature/#71

### LINKED ISSUE #71: Implement Answer Service
-create 
-get

### PR DESCRIPTION
**Pull request by dtymkiv**

### CODE REVIEW

File: src/app/services/answer.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    @staticmethod
+    def get_by_id(answer_id):
+        """
+
+        :param pk:
+        :return: answer with id=pk or None
+
+        """
+        return Answer.query.get(answer_id)
```
Comment (IvanKostiv) [inline src/app/services/answer.py]: For debug it is better to write: 
answer = Answer.query.get(answer_id)
return answer

File: src/app/services/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+"""
+Services init module
+"""
```
Comment (yurdosii) [inline src/app/services/__init__.py]: Why doesn't docstring start from the first line?

File: src/app/services/answer.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+class AnswerService:
+    """ answer service class"""
+
+    @staticmethod
+    def get_by_id(answer_id):
+        """
+
+        :param pk:
+        :return: answer with id=pk or None
+
```
Comment (yurdosii) [inline src/app/services/answer.py]: Extra empty line after ":return: answer with id=pk or None"

File: src/app/services/answer.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        """
+        return Answer.query.get(answer_id)
+
+    @staticmethod
+    def get_by_value(value):
+        """
+
+        :param value:
+        :return: answer with this value or None
+
```
Comment (yurdosii) [inline src/app/services/answer.py]: Extra empty line

File: src/app/services/answer.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
+"""Answer service"""
+
```
Comment (yurdosii) [inline src/app/services/answer.py]: Docstring format:
"""
Answer service
"""

File: src/app/services/answer.py [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+from app import DB
+from app.helper.decorators import transaction_decorator
+from app.models import Answer
+
+
+class AnswerService:
+    """ answer service class"""
```
Comment (yurdosii) [inline src/app/services/answer.py]: Docstring format:
"""
Answer service class
"""

File: src/app/services/answer.py [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+    @staticmethod
+    def get_by_id(answer_id):
+        """
+
+        :param pk:
```
Comment (yurdosii) [inline src/app/services/answer.py]: Method argument isn't pk.

**Review by yurdosii — APPROVED** [review]

**Review by Howrran — APPROVED** [review]

**Review by AndriiKret — APPROVED** [review]

**Review by IvanKostiv — APPROVED** [review]

**Pull request merged**